compact/tracker.xml

```diff
@@ -32,8 +32,8 @@
 
     <comment> Service/Support setup </comment>
     <constant name="VertexSupportCyl1_rmin"          value="VertexBarrel_rmin + 2.1*cm"/>
-    <constant name="VertexSupportCyl2_rmin"          value="VertexBarrel_rmax"/>
-    <constant name="VertexSupportCylCF_thickness"    value="300.0*um" />
+    <constant name="VertexSupportCyl2_rmin"          value="VertexBarrel_rmax-0.3*cm"/>
```

Comment:
0.3 cm is magical constant. Needs naming?

Reply:
done